e2e/cephfs.go

```diff
@@ -67,17 +72,51 @@ var _ = Describe("cephfs", func() {
 
 			By("create and delete a PVC", func() {
 				By("create a PVC and Bind it to an app", func() {
-					pvcPath := cephfsExamplePath + "pvc.yaml"
-					appPath := cephfsExamplePath + "pod.yaml"
 					validatePVCAndAppBinding(pvcPath, appPath, f)
 
 				})
 
 				By("create a PVC and Bind it to an app with normal user", func() {
-					pvcPath := cephfsExamplePath + "pvc.yaml"
 					validateNormalUserPVCAccess(pvcPath, f)
 				})
+
+				By("create/delete multiple PVC and App", func() {
+					totalCount := 2
```

Comment:
may be make it as a variable at global level.

Reply:
I would like to limit the scope of  value within this context, if  required maybe we can have different  variable bound to a particular context to create a different number of PVC's